src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -109,51 +112,55 @@ const SetAclPermissionForm = () => {
 
   return (
     <FormSection
-      title="Set Permission to File"
+      title="Permission for [FileName]"
       state={state}
-      statusType="Permission status"
-      defaultMessage="To be set..."
+      statusType="Status"
+      defaultMessage="No action yet..."
     >
       <Box display="flex" justifyContent="center">
         <form onSubmit={handleAclPermission} autoComplete="off">
-          <FormControl fullWidth>
-            <InputLabel htmlFor="set-acl-to" label="Set permissions to:" />
+          <FormControl required fullWidth sx={{ marginBottom: '1rem' }}>
+            <InputLabel id="permissionType-label">Select One</InputLabel>
+            <Select
+              labelId="permissionType-label"
+              id="permissionType"
+              label="Select One"
+              value={permissionState.permissionType}
+              onChange={(e) =>
+                setPermissionState({ ...permissionState, permissionType: e.target.value })
+              }
+              name="setAclPerms"
+            >
+              <MenuItem value="Give">Give Permission</MenuItem>
+              <MenuItem value="Revoke">Revoke Permission</MenuItem>
+            </Select>
+          </FormControl>
+
+          <FormControl fullWidth sx={{ marginBottom: '1rem' }}>
             <TextField
               id="set-acl-to"
               name="setAclTo"
-              value={username}
-              onChange={(e) => setUsername(e.target.value)}
-              placeholder={username}
-              label="Search Username"
+              value={permissionState.podToSetPermissionsTo}
```

Comment:
2 things:

1. I think it's going to be very difficult for users to remember the pod urls of every person they are interacting with. I think it would be easier for users if they can instead select from a dropdown of people from their userlist. Then we automatically figure out the proper webIds to set.
2. Although we use pod urls for document upload, for permissions we use the webId. So this shouldn't be passing around pod urls, it should be passing around webIds.

Reply:
> 2 things:
> 
> 1. I think it's going to be very difficult for users to remember the pod urls of every person they are interacting with. I think it would be easier for users if they can instead select from a dropdown of people from their userlist (like if this were a 'contact' list like gmail lets you do with known emails). Then we automatically figure out the proper webIds and info to set.
> 2. Although solid uses pod urls for document upload, for permissions solid uses the webId. So this shouldn't be passing around pod urls, it should be passing around webIds.

1. agreed, but where does a list of users come from? Should you not be able to grant anybody you want permission to whatever you want?    If we want a userslist, we would need some way of creating that - like maybe requesting friends or connections as in other social media.... ?    I was under the impression that the webId was not standardized, meaning any server/provider could format it really however they'd like and that's why we were moving to podUrls.

2. If that is the case, then this issue should just be closed? "Change permission forms to use podUrls explicitly, instead of usernames"

Reply:
For 2, you could get WebIds from pod URLs by attaching `profile/card#me` similar to how it's done with messages (see `sendMessageTTL` in session-core). You're still using podUrls explicitly and having them converted into webIds in the process.

Reply:
1. We have userslist.ttl file in the pod, and a UserslistContext in the react front end. Looking over those should answer your question
2. I didn't look over the issue itself too closely, but I think we can just word the issue better. The intent of the issue is good. We don't need a concept of usernames, so we should remove it from the permissions components. I believe the webid is stored for each user in the userlist.

What's the difference between pod urls and web ids? Pod urls for storage, web ids for identity.

Reply:
I added a related issue about enhancing the users list here:

https://github.com/codeforpdx/PASS/issues/351

Reply:
For now the input is asking user to type in the podURL, then that form component converts it to a webID - and sends that as param/argument.

When we switch over to having a dropdown/selection menu we can update however we wish.